sdk/batch/batch-typespec/routes.tsp

```diff
@@ -67,6 +93,8 @@ API.
 }
 
 #suppress "@azure-tools/typespec-azure-core/casing-style" "The names of Model types must use camelCase"
+#suppress "@azure-tools/typespec-azure-core/no-rpc-path-params" "Operations defined using RpcOperation should not have path parameters."
```

Comment:
Is the path parameter needed here for technical reasons or is it a style preference?